src/visualization/components/View.tsx

```diff
@@ -26,6 +28,68 @@ interface Props {
   cellID?: string
 }
 
+interface QueryStatProp {
+  result?: FromFluxResult
+  loading?: RemoteDataState
+}
+
+const QueryStat: FC<QueryStatProp> = ({result, loading}) => {
+  const [queryStart, setQueryStart] = useState(0)
+  const [processTime, setProcessTime] = useState(0)
+  let tableNum = 0
+
+  const tableColumn = result?.table?.getColumn('table')
+  const lastTableValue = tableColumn[tableColumn.length - 1]
+
+  if (typeof lastTableValue === 'string') {
+    tableNum = parseInt(lastTableValue) + 1
+  } else if (typeof lastTableValue === 'boolean') {
+    tableNum = lastTableValue ? 1 : 0
+  } else {
+    // number
+    tableNum = lastTableValue + 1
+  }
+
+  useEffect(() => {
+    if (loading === RemoteDataState.Loading) {
+      // start to count
+      if (queryStart === 0) {
+        setQueryStart(Date.now())
+        setProcessTime(0)
+      }
+      return
+    }
+
+    if (loading === RemoteDataState.Done && queryStart !== 0) {
+      const timePassed = Date.now() - queryStart // ms
+      setQueryStart(0)
+      setProcessTime(timePassed)
+      return
+    }
+
+    setQueryStart(0)
```

Comment:
might wanna use a `useRef` for `queryStart` as you're not expecting it to update the render cycle